weather_mv/loader_pipeline/bq.py

```diff
@@ -0,0 +1,299 @@
+# Copyright 2022 Google LLC
```

Comment:
'bq' as a file name seems terse bordering on cryptic. maybe at least bq_sink, or to_big_query? (Apologies if this is still naming-in-progress and you are still working on the structure, I just saw the emails and got curious)

Reply:
I like the name `bq`, so long as it's obvious that I'm referring to BigQuery. I don't think `_sink` is necessary. According to the Google Python Style Guide, abbreviations are OK so long as they're understood beyond the scope of the project [0]. It seems like "bq" is pretty conventional to mean BigQuery, see [its usage in Pandas](https://pandas.pydata.org/docs/reference/api/pandas.read_gbq.html), for example. 

[0]: https://google.github.io/styleguide/pyguide.html#316-naming